packages/suite/src/components/firmware/FirmwareOffer.tsx

```diff
@@ -71,9 +70,12 @@ export const FirmwareOffer = ({ customFirmware, targetFirmwareType }: FirmwareOf
     const nextVersion = customFirmware
         ? translationString('TR_CUSTOM_FIRMWARE_VERSION')
         : getFwUpdateVersion(originalDevice);
+
+    const isBtcOnly = originalDevice.firmwareType === FirmwareType.BitcoinOnly;
```

Comment:
This needs to take `targetFirmwareType` into account to work properly in firmware type switch flow.

Reply:
![image](https://github.com/user-attachments/assets/82708253-6c94-44ba-a195-5042069c2fcd)
 
 Like this?

Reply:
e240c636ac0c0e6da5c3e609f28e6db6270f21d4

Reply:
I don't have a device here to test with, but this probably won't work correctly during FW installation when switching type. `targetFirmwareType` from props is used in case installation hasn't started yet, while `targetType` is set when installation begins and used in case `targetFirmwareType` prop is not provided. Please test it with e5ebf0e5b373d30e62c681d75528c605a3693a6f.